README.md

```diff
@@ -75,13 +75,15 @@ There are many types of workflow. At FAC, we use [the GitHub flow](https://guide
 ```sh
 $ git branch create-heading-with-shadow
 ```
-
+</br>
+<img src="images/step3(1)GitFlow.png" width="300" height="auto" alt="repo visual after step 1">
 2. Leave the master branch by switching to the new branch you have just created.
 
 ```sh
 $ git checkout create-heading-with-shadow
 ```
-
+</br>
+<img src="images/step3(2)GitFlow.png" width="300" height="auto" alt="repo visual after step 1">
 <!-- ***********************************************************-->
 ## Step 4 - Write enough HTML & CSS to satisfy the requirements
```

Comment:
For some reason in the actual file the step 4 title isn't working :disappointed: 

Here how it looks from my screen:

![image](https://user-images.githubusercontent.com/24490876/35686701-91c20bda-0775-11e8-8159-092a38f6120a.png)